.github/workflows/examples.yml

```diff
@@ -28,4 +28,18 @@ jobs:
       - run: go install -v ./gnovm/cmd/gno
       - run: go run ./gnovm/cmd/gno precompile --verbose ./examples
       - run: go run ./gnovm/cmd/gno build --verbose ./examples
-  # unittests: TODO: matrix with contracts
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: [ "1.19.x", "1.20.x" ]
+        # unittests: TODO: matrix with contracts
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ matrix.go-version }}
+      - uses: actions/checkout@v3
+      - run: go install -v ./gnovm/cmd/gno
+      - run: go run ./gnovm/cmd/gno test --verbose ./examples
```

Comment:
I think we can use the already installed binary (from the previous run step):
`gno test --verbose ./examples`

Reply:
I prefer to keep it as follows: 
1. Using `go run ./gnovm/cmd/gno` is more reliable than relying on the `gno` command, which may vary depending on the environment.
2. Separating the `go install` and `go run` commands helps distinguish between the compilation and testing phases, allowing for distinct reporting of errors and execution time.

Do you think it would be helpful to provide comments explaining the rationale behind this approach?

Reply:
Ah, got it, I don't think there is a need to specifically mention it in the workflow file - I was just curious why the added step of `go run`